enhance_controls.py

```
@@ -0,0 +1,158 @@
+        "grid": "grid/index.md",
+        "treedatagrid": "treedatagrid/index.md",
+        "colorpicker": "colorpicker/colorpicker.md"
+    }
+
+    if name_lower in mappings:
+        potential_path = os.path.join(DOCS_ROOT, mappings[name_lower])
+        if os.path.exists(potential_path):
+            return potential_path
+
+    # Recursive search
+    for root, dirs, files in os.walk(DOCS_ROOT):
```

Ruff (0.14.5)

Loop control variable `dirs` not used within loop body

Rename unused `dirs` to `_dirs`

(B007)